### PR TITLE
Fix #224 Added New Submodule Paste and Indent

### LINKED ISSUE #224: Add paste-and-indent extension to our defaults
This is really helpful when you are pasting code you find on the web into the editor, and want it to have the right indents:

https://github.com/ahuth/brackets-paste-and-indent

Let's add this as a submodule to Bramble, and add it to `ExtensionLoader` for our set of detaultExtensions.

@KeniMorri, can you do this?

### PR DESCRIPTION
**Pull request by KeniMorri**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -37,3 +37,6 @@
 [submodule "src/extensions/default/brackets-browser-livedev"]
 	path = src/extensions/default/brackets-browser-livedev
 	url = https://github.com/humphd/brackets-browser-livedev.git
+[submodule "src/extensions/default/brackets-paste-and-indent"]
+	path = src/extensions/default/brackets-paste-and-indent
+	url = git@github.com:KeniMorri/brackets-paste-and-indent.git
```
Comment (humphd) [inline .gitmodules]: Let's use the original repo vs. your forked URL.  Just send him a PR and he can merge that upstream, and we'll update then.

**Comment by humphd** [comment]
One issue to fix, plus pass travis.

File: .gitmodules (hunk 1 above)
Comment (KeniMorri) [inline .gitmodules]: Alright

**Comment by humphd** [comment]
Looks good, depending on Travis.

**Pull request merged**